sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/options/BlobBeginCopyOptions.java

```diff
@@ -23,7 +23,7 @@
     private Map<String, String> tags;
     private AccessTier tier;
     private RehydratePriority rehydratePriority;
-    private RequestConditions sourceRequestConditions;
+    private BlobRequestConditions sourceRequestConditions;
```

Comment:
I thought you said we were adding a SourceRequestConditions type?

Reply:
Let's keep this PR open until the Arch board meeting today. For now this is synced with .NET but I think we are going to conform with them if making the source request conditions makes sense